server/scripts/participant-stats-rejected.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-console */
+require('dotenv').config({ path: '../.env' });
+const csv = require('fast-csv');
+const path = require('path');
+const { writeFileSync } = require('fs');
+const { dbClient } = require('../db/db.js');
+const { getRejectedParticipantsReport } = require('../services/reporting');
+
+(async () => {
+  if (require.main === module) {
+    try {
+      await dbClient.connect();
+
+      const results = await getRejectedParticipantsReport();
+
+      console.log(results);
+
+      const csvStream = csv.format({ headers: true });
+
+      results.forEach((result) => {
+        csvStream.write({
+          'Participant Name': `${result.participantInfo.firstName} ${result.participantInfo.lastName}`,
```

Comment:
Should remove this PII - all the other reports just key off IDs the program can cross-reference :)